lib/_debugger.js

```diff
@@ -1705,6 +1714,7 @@ Interface.prototype.trySpawn = function(cb) {
     // If it's failed to connect 10 times then print failed message
     if (connectionAttempts >= 10) {
       self.stdout.write(' failed, please retry\n');
+      process.exit(1);
```

Comment:
Seems like an arbitrary change.  Leaving the `return` statement in is a bit odd as well.